jobs/uaa/templates/uaa.yml.erb

```diff
@@ -167,6 +167,10 @@ scim:
     - <%= group %><% end %>
 <% end %>
 
+<% if properties.uaa.scim && properties.uaa.scim.groups then %>
+  groups: <%= properties.uaa.scim.groups %>
+<% end %>
+
```

Comment:
Thanks for the hint! I'm aware of that helper, but wanted the exact similar behaviour as the `external_groups` part which was in the "old" style. Can change it if you like?